powersimdata/scenario/create.py

```diff
@@ -1,5 +1,4 @@
 import copy
-import warnings
 
 import numpy as np
 import pandas as pd
```

Comment:
This is a test comment. Can someone reply so I can see if I get email notifications now?

Reply:
Test

Reply:
Thanks for posting

Reply:
Notifications are now working! For some reason they were being sent to the wrong email address.